packages/app/tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
```

Comment:
This is because your VSCode is not correctly configured.
By default, it uses their default typescript version, which is previous to the one we are using on the project.

Just change the settings for "Use workspace Typescript Version"

Ref: https://github.com/Microsoft/vscode/issues/24719

Reply:
My VSCode was properly configured, to fix the warnings I had to upgrade the Typescript version of the project 🤷‍♂️